doc/source/_toc.yml

```diff
@@ -106,6 +106,8 @@ parts:
           - file: workflows/package-ref
       - file: ray-more-libs/index
         title: More Ray ML Libraries
+      - file: ray-air/getting-started
```

Comment:
How about we nest it under "More Ray ML Libraries" for now?

Reply:
that's actually a good idea. makes it easier to surface stuff later

Reply:
done